internal/config/client.go

```diff
@@ -175,3 +183,18 @@ func (c *MongoDBClient) GetRealmClient(ctx context.Context) (*realm.Client, erro
 
 	return realmClient, nil
 }
+
+func (c *Config) appendToUserAgent(additionalInfo string) {
+	currentUA := c.UserAgent
+	if !strings.Contains(currentUA, additionalInfo) {
```

Comment:
why do we need to check if it's already there? is the code being called more than once?

Reply:
>is the code being called more than once?

no, but was trying to keep the method generic